### DISCUSSION TITLE
v1.17.0

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#520] [Release]
# Arc Cannon and Double Barrel Shotgun

![cropped-northstar-arc-cannon](https://github.com/R2Northstar/Northstar/assets/40122905/650e4780-04b1-4de8-bb4f-7b457df9286c)

![double-barrel-cropped](https://github.com/R2Northstar/Northstar/assets/40122905/83ac7768-3541-4ded-87a9-95d79fb80aa1)

**Features:**
Add Arc cannon ([Mods#600](https://github.com/R2Northstar/NorthstarMods/pull/600))
Add cut Doublebarrel shotgun by using model from Titanfall Online ([Mods#660](https://github.com/R2Northstar/NorthstarMods/pull/660))
Add support for loading Thunderstore mods natively ([Launcher#503](https://github.com/R2Northstar/NorthstarLauncher/pull/503))
Search for Northstar.dll in profile directory ([Launcher#451](https://github.com/R2Northstar/NorthstarLauncher/pull/451))
Add ability to disable executions via a Callback ([Mods#633](https://github.com/R2Northstar/NorthstarMods/pull/633))
Add OS, Wine and Proton logging ([Launcher#502](https://github.com/R2Northstar/NorthstarLauncher/pull/502))

**Fixes:**
Fix grunt death callbacks ([Mods#632](https://github.com/R2Northstar/NorthstarMods/pull/632))

**Build system:**
CMAKE: set build type if not previously set, turn bin dir message into status ([Launcher#497](https://github.com/R2Northstar/NorthstarLauncher/pull/497))

**Contributors:**
@ASpoonPlaysGames @EM4Volts @F1F7Y @GeckoEidechse @Jan200101 @Masterliberty @Neoministein

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.17.0'>v1.17.0</a>.</em>

**Comment by titanbetter** [comment]
how do you update the game

**Reply by itscynxx** [reply]
i replied to your issue on the repo for this

please read that
